include/imgui-ws/imgui-ws.h

```diff
@@ -81,4 +84,6 @@ class ImGuiWS {
     private:
         struct Impl;
         std::unique_ptr<Impl> m_impl;
+	THandler connect_handler = nullptr;
+	THandler disconnect_handler = nullptr;
```

Comment:
Use spaces instead of tabs:

```suggestion
        THandler connect_handler = nullptr;
        THandler disconnect_handler = nullptr;
```

Reply:
It should be all spaces now. Sorry about that. [EditorConfig](https://editorconfig.org/) may be helpful here in the future.